common/include/Logger.h

```diff
@@ -0,0 +1,108 @@
+#ifndef LOGGER_H_
```

Comment:
this header file needs a copyright assignment as all other header files in the project